db/virtual_tables.cc

```diff
@@ -68,6 +68,8 @@ class cluster_status_table : public memtable_filling_virtual_table {
             .with_column("peer", inet_addr_type, column_kind::partition_key)
             .with_column("dc", utf8_type)
             .with_column("up", boolean_type)
+            .with_column("drained", boolean_type)
```

Comment:
I assume we don't care about the fact that during upgrade new nodes show more columns than the old ones, do we?